composer.json

```diff
@@ -75,6 +75,7 @@
         "symfony/css-selector": "~2.8",
         "drupal/config_installer": "^1.3",
         "geerlingguy/drupal-vm": "^4.2",
+        "harvesthq/chosen": "^1.0",
```

Comment:
404 https://packagist.org/search/?q=harvesthq%2Fchosen&reason=package_not_found

Should be components/chosen